src/main/antlr4/imports/mysql_alter_table.g4

```diff
@@ -65,3 +65,9 @@ ignored_alter_specifications:
     ;
   algorithm_type: DEFAULT | INPLACE | COPY;
   lock_type: DEFAULT | NONE | SHARED | EXCLUSIVE;
+
+alter_ordering: alter_ordering_column (ASC|DESC)?;
+alter_ordering_column:
+    name '.' name '.' name
```

Comment:
thought that table_name would take care of tbl and db.tbl, putting the logic in one place... think it's a small thing though, pr looks just fine to me